fibers.texi

```diff
@@ -605,7 +605,7 @@ outside of a fiber.
 There is also a low-level constructor for other modules that implement
 primitive operation types:
 
-@defun make-base-operation wrap-fn try-fn block-fn
+@defun make-base-operation wrap-fn try-fn block-fn [cancel-fn]
```

Comment:
This naming is misleading -- there still is no mechanism for cancelling a fiber performing an operation (as in, something like `cancel-thread` or POSIX cancellation).

Reply:
Also documentation for `cancel-fn` is missing. (The lack of documentation for the other arguments doesn't make the lack of documentation for `cancel-fn` fine.)

Reply:
I guess there are arguments for and against this name, but in this particular case, the timer is indeed *canceled*.  I've kept this name in the new version.